### PR TITLE
use the new license generator

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

File: generate_licenses.sh [hunk 1]
```diff
@@ -2,91 +2,16 @@
...
-    product="true"
-    ;;
-  *)
-    product="false"
-    ;;
-esac
+dir=$(pwd)
 
+cd $GENERATOR_HOME
+mvn clean package -Dexample.project.dir="$dir" -Dexternal.license.service="$LICENSE_SERVICE"
```
Comment (sberyozkin) [inline generate_licenses.sh:10]: That will work without the external service as well, right ?

**Review by sberyozkin — APPROVED** [review]
Looks good :-)

File: generate_licenses.sh (hunk 1 above)
Comment (Ladicek) [reply generate_licenses.sh:10]: Yes: https://github.com/thorntail-examples/booster-license-generator/blob/master/src/main/java/io/thorntail/examples/license/generator/ThorntailGeneratorProperties.java#L34

**Comment by Ladicek** [comment]
Thank you!

**Pull request merged**